src/py/flwr/server/run_serverapp.py

```diff
@@ -107,7 +107,7 @@ def run_server_app() -> None:
     run(server_app_attr, driver, server_app_dir)
 
     # Clean up
-    del driver
+    driver.__del__()
```

Comment:
This is a temporary fix. The fix of this problem is outside the scope of this PR.

The `del driver` will not trigger `driver.__del__` here, b/c there may be other references to the driver object. If the driver object is not cleaned up here, it will then cause an error in telemetry. I think we should fix this issue.